cyraft/node.py

```diff
@@ -685,12 +717,22 @@ async def _on_term_timeout(self) -> None:
             self._term,
         )
         assert self._state == RaftState.LEADER, "Only leaders have a term timeout"
-        self._term += 1
```

Comment:
As far as I can tell, this is the only relevant change, all other changes are related to leader election.

(Not talking about small fixes/log changes)